src/styles/less/Header.less

```diff
@@ -1,19 +1,30 @@
 @import '~antd/dist/antd.less';
 @import '~antd/lib/style/themes/default.less';
 
+
 .hero {
   background-image: url('../../assets/images/cityscape.svg');
-  min-height: 30vh;
+  min-height: 45vh;
   background-repeat: no-repeat;
   background-size: 100% auto;
   background-position: bottom;
   border: 5px solid @header-nero;
   background-color: @turquoise;
   position: relative;
-
+  
   @media @mobile {
-    min-height: 20vh;
+    min-height: 25vh;
+  }
+  @media @tablet {
+    min-height: 40vh;
+  }
+  @media (max-width: 650px) {
+    min-height: 25vh;
+  }
+  @media (max-width: 768px) {
+    min-height: 25vh;
   }
+  
```

Comment:
Great use of the various media queries! I remember discussing the extra “white space” under the text on smaller screen sizes during a recent design critique. I don’t know if that is something we need to necessarily change on this PR since we may need stakeholder feedback on it. The fix for the text is excellent, however!